### ISSUE TITLE
Installing the package takes more than 20 minutes still loading, Using Both npm or yarn

### ISSUE THREAD

**Issue by mustafaskyer** [react-native-mapbox-gl/maps#390]
**Describe the bug**
Installing the package takes more than 20 minutes still loading, Using Both npm or yarn

**To Reproduce**
yarn add @react-native-mapbox-gl/maps

**Expected behavior**
success installation.

**Screenshots**
![Screen Shot 2019-09-18 at 9 30 48 PM](https://user-images.githubusercontent.com/46998415/65175595-cc8a3700-da5b-11e9-9843-b4fdd2703545.png)

**Versions (please complete the following information):**
 - Platfrom: iOS
 - Device: iPhoneX
 - OS: MAC
 - SDK Version 28.0.3
 - React Native Version 0.59.10

**Additional context**

**Comment by kristfal** [comment]
This is happening if you have a poor network connection or there is an issue with Mapbox’s CDN.

I’d suggest you try on a different network or just wait a bit. You can try to ping the iOS SDK or download it directly by looking at the scripts triggered in package.json

**Comment by mustafaskyer** [comment]
@kristfal  Thanks for your fast reply,
I tried with different network and got same issue
my network speed is **Download** 78.6 Mbps, **upload**: 18 Mbps

**Comment by mustafaskyer** [comment]
also i tried to run script

` npm run fetch:ios:sdk `

it took a while & not done yet

![Screen Shot 2019-09-18 at 10 09 15 PM](https://user-images.githubusercontent.com/15132084/65178430-5ee10980-da61-11e9-9083-084e14d4200c.png)

**Comment by sacheen** [comment]
looks like it is the CDN / amazon .. also have a 100Mbps connection 
takes forever to download. 
https://mapbox.s3.amazonaws.com/mapbox-gl-native/ios/builds/mapbox-ios-sdk-5.3.2-dynamic.zip

I think it eventually times out, shows as completed but the framework does not unzip.
Then you can't compile because the framework is missing.

**Comment by sacheen** [comment]
confirmed time out / connection issues.

Print: Entry, ":MGLSemanticVersionString", Does Not Exist
curl: (56) LibreSSL SSL_read: SSL_ERROR_SYSCALL, errno 54
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of temp.zip or
        temp.zip.zip, and cannot find temp.zip.ZIP, period.
mv: rename temp/dynamic/Mapbox.framework to ./Mapbox.framework: No such file or directory
rm: temp: No such file or directory

**Comment by lmyEsther** [comment]
Has anyone resolved this issue? Please!

**Comment by mustafaskyer** [comment]
hello,
for me it was because of slow connection,
try another better internet connection solved it ✅

**Comment by kristfal** [comment]
There is nothing we can do here really. We are just downloading the official mapbox SDK from the official source. 

Please post an [upstream](https://github.com/mapbox/mapbox-gl-native) ticket if you want someone from Mapbox to review this.

**Comment by mfazekas** [comment]
@kristfal actually using cocoapods deps would solve this issue i think. See https://github.com/react-native-mapbox-gl/maps/pull/535#issuecomment-561487866

**Comment by CristianHatz** [comment]
This is still happening for me.

I am using `react-native 0.59.2` and I need to use `react-native-mapbox-gl/maps 7.0.9`(can't upgrade to RN 0.60+ yet). 

Both `npm install` and `yarn add` are stucked in `Building fresh packages` phase. 

I have checked the install scripts and noticed that the Amazon s3 download link returns AccesDenied: https://mapbox.s3.amazonaws.com/mapbox-gl-native/ios/builds/mapbox-ios-sdk-7.0.9-dynamic.zip

**Comment by CristianHatz** [comment]
I forked the repo and checkout the 7.0.9 branch. I changed in 'maps/scripts/download-mapbox-gl-native-ios.sh' the 'curl -sS https://mapbox.s3.amazonaws.com/mapbox-gl-native/ios/builds/mapbox-ios-sdk-$VERSION-dynamic.zip > temp.zip' with 'curl -sS https://mapbox.s3.amazonaws.com/mapbox-gl-native/ios/builds/mapbox-ios-sdk-5.5.0-dynamic.zip > temp.zip' . And change the package.json to get the lib from my repo. It took 5 minutes to download but worked for me.
